nucliadb_utils/nucliadb_utils/audit/stream.py

```diff
@@ -175,12 +180,34 @@ async def search(
         # Search is a base64 encoded search
         auditrequest = AuditRequest()
         auditrequest.origin = origin
+        auditrequest.client_type = client_type  # type: ignore
         auditrequest.userid = user
         auditrequest.kbid = kbid
         auditrequest.search.CopyFrom(search)
         auditrequest.timeit = timeit
         auditrequest.resources = resources
         auditrequest.type = AuditRequest.SEARCH
         auditrequest.time.FromDatetime(datetime.now())
+        auditrequest.trace_id = get_current_span().get_span_context().trace_id
+        await self.send(auditrequest)
+
+    async def suggest(
+        self,
+        kbid: str,
+        user: str,
+        client_type: int,
+        origin: str,
+        timeit: float,
+    ):
+        # Search is a base64 encoded search
```

Comment:
this comment does not belong here right?

Reply:
yup, copy paste

Reply:
Fixed